### DISCUSSION TITLE
Unable to accept invitation

### DISCUSSION THREAD

**Discussion by timdrijvers** [community/community#52076] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I'm accepting an invitation to an organisation, this raises the unicorn page "This page is taking too long to load." after a while. In the meantime, i'm getting emails that i'm added to the repos in the organisation. When i visit the projects, i dont have access.

**Comment by meppu** [comment]
Try another way to accept an invite, go to "Settings" and then click "Organizations" ([quick link](https://github.com/settings/organizations)) and you can manage all organizations from this page. Just accept the invite from here (screenshot is from another person):

![image](https://user-images.githubusercontent.com/112294217/230148001-0bc73e55-49d0-4de0-b84f-93818d12b753.png)
